service-commands/scripts/A

```diff
@@ -1,3 +1,301 @@
-#!/bin/bash
-cd $PROTOCOL_DIR/service-commands
-node scripts/setup.js $@
\ No newline at end of file
+#!/usr/bin/env python3
+
+import os
+import sys
+import subprocess
+import argparse
+
+
+def create_instance(name, image_project, image_family, size, machine_type):
+    project = (
+        subprocess.run(
+            [
+                "gcloud",
+                "config",
+                "get-value",
+                "project",
+            ],
+            capture_output=True,
+        )
+        .stdout.decode()
+        .strip()
+    )
+
+    account = (
+        subprocess.run(
+            ["gcloud", "config", "list", "account", "format", "value(core.account)"],
+            capture_output=True,
+        )
+        .stdout.decode()
+        .strip()
+    )
+
+    print("Project:", project)
+    print("Account:", account)
+
+    input("Press Enter to confirm account and project...")
```

Comment:
Wouldn't this continue anyway no matter what the user pressed? Except like cmd+c

Reply:
Yes, that's right, I can change this